templates/Web/_composition/Vue/Page.Vue.AddHome/src/router/index_postaction.js

```diff
@@ -0,0 +1,8 @@
+﻿export default new VueRouter({
+  routes: [
+    //^^
+    //{[{
+    { path:"/", redirect: "/Param_HomePageName" }
+    //}]}
+  ]
+});
```

Comment:
I'm comparing the structure to the template for React, so I'm wondering why VueRouter is required? It seems like for React, we are not exporting any ReactRouter.

Reply:
Oh haha good question its done differently in vue, that's the main reason.

Look at this guide: https://medium.com/codingthesmartway-com-blog/vue-js-routing-with-vue-router-4c428fabb078

and these docs for more info! https://router.vuejs.org/guide/#javascript

Then check this file out:
https://github.com/microsoft/WebTemplateStudio/blob/5306b002acaa62349bf03904482730212f2e48fd/templates/Web/Projects/VueDefault/src/main.js

I completely understand the confusion here though! Hope these resources help!